src/Microsoft.DotNet.XHarness.CLI/Commands/iOS/iOSTestCommand.cs

```diff
@@ -28,337 +22,129 @@ namespace Microsoft.DotNet.XHarness.CLI.Commands.iOS
     /// <summary>
     /// Command which executes a given, already-packaged iOS application, waits on it and returns status based on the outcome.
     /// </summary>
-    internal class iOSTestCommand : TestCommand
+    internal class iOSTestCommand : iOSAppCommand
     {
-        private const string CommandHelp = "Runs a given iOS/tvOS/watchOS application bundle in a target device/simulator";
-
-        private static readonly string s_mlaunchLldbConfigFile = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.Personal), ".mtouch-launch-with-lldb");
+        private const string CommandHelp = "Runs a given iOS/tvOS/watchOS test application bundle containing TestRunner in a target device/simulator";
 
         private readonly iOSTestCommandArguments _arguments = new iOSTestCommandArguments();
-        private readonly ErrorKnowledgeBase _errorKnowledgeBase = new ErrorKnowledgeBase();
-        private bool _createdLldbFile;
 
         protected override string CommandUsage { get; } = "ios test [OPTIONS] [-- [RUNTIME ARGUMENTS]]";
         protected override string CommandDescription { get; } = CommandHelp;
-        protected override TestCommandArguments TestArguments => _arguments;
-
-        public iOSTestCommand() : base(CommandHelp)
-        {
-        }
-
-        protected override async Task<ExitCode> InvokeInternal(ILogger logger)
-        {
-            var processManager = new MlaunchProcessManager(_arguments.XcodeRoot, _arguments.MlaunchPath);
-            var deviceLoader = new HardwareDeviceLoader(processManager);
-            var simulatorLoader = new SimulatorLoader(processManager);
-
-            var logs = new Logs(_arguments.OutputDirectory);
-
-            var cts = new CancellationTokenSource();
-            cts.CancelAfter(_arguments.Timeout);
-
-            var exitCode = ExitCode.SUCCESS;
-
-            foreach (var target in _arguments.TestTargets)
-            {
-                var tunnelBore = (_arguments.CommunicationChannel == CommunicationChannel.UsbTunnel && !target.Platform.IsSimulator())
-                    ? new TunnelBore(processManager)
-                    : null;
-                var exitCodeForRun = await RunTest(logger, target, logs, processManager, deviceLoader, simulatorLoader,
-                    tunnelBore, cts.Token);
-
-                if (exitCodeForRun != ExitCode.SUCCESS)
-                {
-                    exitCode = exitCodeForRun;
-                }
-            }
-
-            return exitCode;
-        }
+        protected override iOSRunCommandArguments iOSRunArguments => _arguments;
 
-        private static bool IsLldbEnabled() => File.Exists(s_mlaunchLldbConfigFile);
-
-        private void NotifyUserLldbCommand(ILogger logger, string line)
+        public iOSTestCommand() : base("test", true, CommandHelp)
```

Comment:
```suggestion
        public iOSTestCommand() : base("test", false, CommandHelp)
```

Reply:
This was actually `true` before and needs to be `true`

Reply:
I double check it and still it seems to me that it was false by default. Please correct me if I'm wrong:
![image](https://user-images.githubusercontent.com/19972657/100093010-3a4ba300-2e57-11eb-99ed-8492771b2785.png)

Reply:
You're right! I am sorry, it was the WASM command that has it set to true. I didn't check it properly and thought the base class was setting this to `true`. Fixed